addons/XRHandtrackingTools/pose_records.tres

```diff
@@ -994,6 +994,7 @@ bone_poses = {
 }
 
 [resource]
+resource_local_to_scene = true
```

Comment:
really necessary?

Reply:
Wanted to try it (does it matter? =

Reply:
Not that much i guess. Just duplicates the resource ergo more memory usage. It doesn't harm anything. (Yet XD)